pkg/network/istio/consts.go

```diff
@@ -1,4 +1,4 @@
-package consts
+package istio
```

Comment:
call this file annotations, the fact that there are const in side is not relevant and open the door to place here rnadom istio stuff.

Reply:
Done along with the rebase, Thanks!